guides/doc-Creating_a_Standard_Operating_Environment/master.adoc

```diff
@@ -0,0 +1,18 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: soe
+:ProductName: {ProjectServer}
+
+= {SOEDocTitle}
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+include::common/assembly_administering-project-with-projectwebui.adoc[leveloffset=+1]
+
+include::common/assembly_administering-project-with-hammer.adoc[leveloffset=+1]
```

Comment:
How about making this into a single chapter, e.g. "Interfacing with Project", and including API and webhooks? As in "what ways there are to get data in and out?".